### ISSUE TITLE
Ellipses dawn with create_ellipse_filled() are different size than those drawn with draw_ellipse_filled()

### ISSUE THREAD

**Issue by SirGnip** [pythonarcade/arcade#756]
### Actual behavior:
A shape created with create_ellipse_filled() and put in a `ShapeElementList` is a different size than a shape drawn with `draw_ellipse_filled()`, even though they have the same width/height

### Expected behavior:
I expect both functions to draw the same size ellipse.

### Steps to reproduce/example code:

The ellipses are different sizes even though they have the same width and height:

![image](https://user-images.githubusercontent.com/5678175/93670218-703b6800-fa5f-11ea-80dc-ab2278f8cdd0.png)

```
import arcade

x = 200
y = 100
oval_width = 50
oval_height = 100


class MyGame(arcade.Window):
    def __init__(self, width, height, title):
        super().__init__(width, height, title)
        shape = arcade.create_ellipse_filled(x, y, oval_width, oval_height, arcade.color.RED)
        self.shape_list = arcade.ShapeElementList()
        self.shape_list.append(shape)

    def on_draw(self):
        arcade.start_render()
        self.shape_list.draw()
        arcade.draw_ellipse_filled(x, y + 200, oval_width, oval_height, arcade.color.BLUE)

def main():
    MyGame(400, 400, "size repro")
    arcade.run()


if __name__ == "__main__":
    main()
```

**Comment by SirGnip** [comment]
Example above is from v2.4.2

**Comment by einarf** [comment]
oh. That is probably a shader bug. Will look into it.

**Comment by pvcraven** [comment]
The ellipses had a radius/diameter mix up a while back. Looks like the fix didn't get into the 'create' methods.

**Comment by einarf** [comment]
What is the right size then? Is size a radius or diameter? To me it seems like diameter in api docs because it seems to refer to the entire width and height... or am I crazy?

**Comment by SirGnip** [comment]
My assumption was that the height and width of an ellipse refer to the full diameter of the ellipse on the X and Y axes.  I think of it as the ellipse is drawn inside a rectangle defined by height and width.

**Comment by einarf** [comment]
It seems the unbuffered shapes (`draw_`) have unchanged behavior between 2.3.15 and 2.4.2. I'm guessing this is more about what version we should break? 😄

**Comment by pvcraven** [comment]
The create_ellipse one needs to be updated looks like.

**Comment by pvcraven** [comment]
Should be fixed in 2.5.1 scheduled to be pushed today.
